float64 max_fwd_velocity
float64 max_rev_velocity
float64 min_fwd_velocity
float64 max_accel
float64 max_decel
float64 max_ang_velocity
float64 max_ang_accel
float64 max_lateral_accel
float64 vehicle_length
float64 vehicle_width
float64 stopping_distance_1M
float64 lidar_spacing
float64 laser_fov
int8 vehicle_gear
int8 curve_type
float64 goal_threshold 
float64 orientation_corr_threshold
float64 mpc_horizon
float64 min_lookahead
float64 max_lookahead
float64 horizon_percent_change
float64 lookahead_smoother
float64 lookahead_factor
float64 curvature_slowdown
float64 curvature_slowdown_multiplier
---
bool result
string information
